packages/dm-core-plugins/blueprints/table/TableFunctionalityConfig.json

```
@@ -0,0 +1,48 @@
+{
+  "type": "CORE:Blueprint",
+  "name": "TableFunctionalityConfig",
+  "attributes": [
+    {
```

an attribute with name "type" is required for blueprints :) 

```suggestion
    {
          "attributeType": "string",
          "type": "CORE:BlueprintAttribute",
          "name": "type",
    }
```

---

decided not necessary